superset/assets/src/visualizations/nvd3_vis.js

```diff
@@ -866,10 +866,19 @@ export default function nvd3Vis(slice, payload) {
           .attr('width', width)
           .call(chart);
 
+        if (vizType === 'line' && fd.annotation_layers && fd.annotation_layers.length > 0) {
+          fd.annotation_layers.forEach(annotatedLayer => {
+            if (annotatedLayer.hasOwnProperty('markerWidth')) {
```

Comment:
This will return true for all annotation layer as  We have added this as a default prop.

We should filter on annotation_source

Reply:
done, added check with annotation_type